pcsx2/CMakeLists.txt

```
@@ -738,6 +738,13 @@ endif()
 
 add_pcsx2_executable(${Output} "${pcsx2FinalSources}" "${pcsx2FinalLibs}" "${pcsx2FinalFlags}")
 
+if(NOT USE_CLANG)
+	if(${CMAKE_VERSION} VERSION_GREATER_EQUAL "3.16")
```

```suggestion
	if(COMMAND target_precompile_headers)
```

> if(COMMAND command-name)
    True if the given name is a command, macro or function that can be invoked.

Source: https://cmake.org/cmake/help/v3.0/command/if.html